src/server/apis/webhook.ts

```diff
@@ -38,7 +41,87 @@ function getRouter(): express.Router {
     }
   });
 
+  webhookRouter.post('/:action', async (request: express.Request, response: express.Response) => {
+    const loginDetails = await getLoginFromRequest(github, request);
+    if (!loginDetails) {
+        response.sendStatus(400);
+        return;
+    }
+
+    if (!request.body.org) {
+      response.status(400).send('No org provided.');
+      return;
+    }
+
+    if (request.params.action === 'add') {
+      const githubAPIResponse = await fetch(
+        `https://api.github.com/orgs/${request.body.org}/hooks`,
+        {
+          method: 'POST',
+          headers: {
+            'Authorization': `token ${loginDetails.token}`,
+            'User-Agent': 'project-health',
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            name: 'web',
+            active: true,
+            events: [
+              '*',
+            ],
+            config: {
+              url: 'https://project-health-internal.googleplex.com/api/webhook',
```

Comment:
What's our plan here?

Reply:
Moving to public - will keep as this for now just to carry on work and will have to change at a later stage.